lib/ansible/modules/cloud/azure/azure_rm_storageaccount_facts.py

```diff
@@ -103,170 +104,234 @@
                 - The access tier for this storage account.
+            returned: always
+            type: str
             sample: Hot
         https_only:
             description:
-                -  Allows https traffic only to storage service if sets to true.
+                -  Allows https traffic only to storage service if sets to C(true).
+            returned: always
+            type: bool
             sample: false
         provisioning_state:
             description:
                 - Gets the status of the storage account at the time the operation was called.
-                - Possible values include 'Creating', 'ResolvingDNS', 'Succeeded'.
+                - Possible values include C(Creating), C(ResolvingDNS), C(Succeeded).
+            returned: always
+            type: str
             sample: Succeeded
         secondary_location:
             description:
                 - Gets the location of the geo-replicated secondary for the storage account.
-                - Only available if the accountType is Standard_GRS or Standard_RAGRS.
+                - Only available if the I(account_type=Standard_GRS) or I(account_type=Standard_RAGRS).
+            returned: always
+            type: str
             sample: westus
         status_of_primary:
             description:
-                - Gets the status indicating whether the primary location of the storage account is available or unavailable.
+                - Gets the status indicating whether the primary location of the storage account is C(available) or C(unavailable).
+            returned: always
+            type: str
             sample: available
         status_of_secondary:
             description:
-                - Gets the status indicating whether the secondary location of the storage account is available or unavailable.
+                - Gets the status indicating whether the secondary location of the storage account is C(available) or C(unavailable).
+            returned: always
+            type: str
             sample: available
         primary_location:
             description:
                 - Gets the location of the primary data center for the storage account.
+            returned: always
+            type: str
             sample: eastus
         primary_endpoints:
             description:
-                - Gets the URLs that are used to perform a retrieval of a public blob, queue, or table object.
-                - Note that Standard_ZRS and Premium_LRS accounts only return the blob endpoint.
+                - Gets the URLs that are used to perform a retrieval of a public I(blob), I(queue), or I(table) object.
+                - Note that C(Standard_ZRS) and C(Premium_LRS) accounts only return the blob endpoint.
+            returned: always
             type: complex
             contains:
                 blob:
                     description:
                         - Gets the primary blob endpoint and connection string.
```

Comment:
I'd remove "Gets" from all of these descriptions, so they would read "The primary blob endpoint and connection string." etc.